src/main/java/io/relution/jenkins/awssqs/factories/SQSFactoryImpl.java

```diff
@@ -94,8 +99,22 @@ public SQSChannel createChannel(final io.relution.jenkins.awssqs.interfaces.SQSQ
     private ClientConfiguration getClientConfiguration(final io.relution.jenkins.awssqs.interfaces.SQSQueue queue) {
         final ClientConfiguration config = new ClientConfiguration();
 
-        // TODO Add support for proxy
-
+        ProxyConfiguration proxyConfig = Jenkins.getInstance().proxy;
```

Comment:
This is the line Jenkins Build is complaining about. I'd try adding a null check on the `Jenkins.getInstance()` method.